### ISSUE TITLE
Empty CDS queue after download

### ISSUE THREAD

**Issue by khufkens** [bluegreen-labs/ecmwfr#110]
@eliocamp  `wf_delete()` functionality missing from new R6 services.

I merged the previous branch after fixing docs and the dreaded debugging of the unit tests.

**Comment by khufkens** [comment]
Seems functional as of #111

**Comment by eliocamp** [comment]
Looks good.

**Comment by khufkens** [comment]
Yeah, close to wrapping this up for CRAN. Will be a nice addition supporting some use cases which have been out of reach or impractical for me (gathering driver data for ecosystem models at the point / location scale - globally).
